api/tests/routes/pro/get_suggested_subcategories_test.py

```diff
@@ -0,0 +1,46 @@
+from unittest import mock
+
+import pytest
+
+from pcapi.core import testing
+from pcapi.core.external.subcategory_suggestion_backends import fixtures
+from pcapi.core.external.subcategory_suggestion_backends.subcategory_suggestion import SubcategorySuggestionBackend
+import pcapi.core.offerers.factories as offerers_factories
+import pcapi.core.users.factories as users_factories
+
+
+@pytest.mark.usefixtures("db_session")
+class Returns401Test:
+    def test_access_by_no_one(self, client):
+        response = client.get("/offers/suggested-subcategories?offer_name=foo")
+
+        assert response.status_code == 401
+
+
+@pytest.mark.usefixtures("db_session")
+@mock.patch("pcapi.core.external.subcategory_suggestion.subcategory_suggestion_backend", SubcategorySuggestionBackend())
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏même remarque, utilise le TestingBackend (j'arrête de répéter 😛)

Reply:
on préfère utiliser le bon `backend` pour faire le test plutôt que le mock, qui lui est testé à part